packages/web-fragments/test/playground/script-insertion/index.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
+<html>
+	<head>
+		<script type="module">
+			import { initializeWebFragments } from 'web-fragments';
+			initializeWebFragments();
+		</script>
+	</head>
+	<body>
+		<h1>WF Playground: Inert script evaluation</h1>
+
+		<web-fragment-host fragment-id="script-insertion">
```

Comment:
I really don't think we should be writing tests in this way - using the web-fragment-host directly, as it relies on lots of implementation details that will make this test fragile and possibly not a good representation of the real application code.

Is there a reason why you didn't create a `fragment.html` file like all the other tests do?

I'll be happy to help you rewrite this test if needed.

Reply:
We can rewrite this test once the PR for enabling piercing in the fragment test lands